db/migrate/20201012120000_update_vms_table_comment.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateCommentToVmsTableFormat < ActiveRecord::Migration[5.0]
+  def up
+    change_column_comment :vms, :format, "The format of the VM's disk, such as tier1."
```

Comment:
`tier1` isn't clear to me...perhaps these are more/better examples.  @agrare ?

Reply:
Let me give some context to explain my approach (and maybe you'll tell I'm trying to push a square peg through a round hole).

In IBM Cloud PowerVS, VMs have an attribute that determines which type of volume can be attached.  We're writing this volume type in this `vms` table `format` column. Also, this type corresponds to `CloudVolumeType` stored in the `cloud_volume_types` DB table. We haven't implemented any of it yet, but in the future we'll want to use this data for things like filtering the list of VMs shown in the attach volume interface.

Since it's been marked deprecated I'm assuming no other providers make use of it, so I changed the example name to one of the types we use.

Reply:
Some of the other options would be `vmdk, qcow2, raw`

Reply:
> but in the future we'll want to use this data for things like filtering the list of VMs shown in the attach volume interface.

Typically we do this with an "allowed_cloud_volume_types" method (as a part of the provisioning workflow), but yeah, I could see under the covers that method using this column.